odd-platform-ui/src/components/Overview/DataEntityList/DataEntityList.tsx

```diff
@@ -23,39 +23,53 @@ const DataEntityList: React.FC<OverviewDataEntityProps> = ({
   const { dataEntityDetailsPath } = useAppPaths();
 
   return (
-    <Grid item>
+    <S.DataEntityListContainer item>
       <S.SectionCaption variant="h4" sx={{ mb: 2 }}>
         {entityListIcon}
         {entityListName}
       </S.SectionCaption>
+
       <S.ListLinksContainer>
         {dataEntitiesList.map(item => (
           <li key={item.id}>
-            <Grid container alignItems="center" wrap="nowrap">
-              <S.ListLink to={dataEntityDetailsPath(item.id)}>
+            <S.ListLink
+              to={dataEntityDetailsPath(item.id)}
+              className={item.hasAlerts ? 'has-alerts' : undefined}
+            >
+              <Box
+                display="flex"
+                flexWrap="nowrap"
+                alignItems="center"
+                overflow="hidden"
+              >
```

Comment:
Let's agree that if we have got styles file and number of style props more than 4 - we will wrap it to styled-component.